client/hotrod-client/src/test/java/org/infinispan/client/hotrod/multimap/RemoteMultimapCacheAPITWithDuplicatesTest.java

```diff
@@ -0,0 +1,58 @@
+package org.infinispan.client.hotrod.multimap;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.client.hotrod.test.HotRodClientTestingUtil;
+import org.infinispan.client.hotrod.test.InternalRemoteCacheManager;
+import org.infinispan.client.hotrod.test.SingleHotRodServerTest;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+
+import static org.testng.AssertJUnit.assertTrue;
+import static org.testng.AssertJUnit.assertEquals;
+
+@Test(groups = "functional", testName = "client.hotrod.RemoteMultimapCacheAPITWithDuplicatesTest")
+public class RemoteMultimapCacheAPITWithDuplicatesTest extends SingleHotRodServerTest {
+
+    private RemoteMultimapCache<String, String> multimapCache;
+
+    @Override
+    protected RemoteCacheManager getRemoteCacheManager() {
+        ConfigurationBuilder builder = HotRodClientTestingUtil.newRemoteConfigurationBuilder();
+        // Do not retry when the server response cannot be parsed, see ISPN-12596
+        builder.forceReturnValues(isForceReturnValuesViaConfiguration()).maxRetries(0);
+        builder.addServer().host("127.0.0.1").port(hotrodServer.getPort());
+        return new InternalRemoteCacheManager(builder.build());
+    }
+
+    @Override
+    protected void setup() throws Exception {
+        cacheManager = createCacheManager();
+        hotrodServer = createHotRodServer();
+        remoteCacheManager = getRemoteCacheManager();
+        remoteCacheManager.start();
+        MultimapCacheManager<String, String> rmc = RemoteMultimapCacheManagerFactory.from(remoteCacheManager);
+        // TODO: Avoid conflict names with a namespace or configuration between cache and multimap cache
```

Comment:
You can just use the class name of the test for the cache name.

Reply:
So it would be `RemoteMultimapCacheTest`?